Business resume/contact.html

```diff
@@ -0,0 +1,113 @@
+    <link rel="stylesheet" href="contact.css">
+    <link rel="stylesheet" href="resume.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Business Resume</title>
+</head>
+<body>
+    <header>
+        <div class="header-left">
+            <div class="header-box">
+                <div class="square">
+                </div>
+                <a href="index.html">Maya Nelson</a>
+            </div>
+            
+            <span>/</span>
+            
+            <a class="title-header" href="index.html">PROJECT MANAGER</a>
+        </div>
+
+        <div class="header-right">
+            <input id="toggle" type="checkbox"></input>
+
+            <label for="toggle" class="hamburger">
+            <div class="top-bun"></div>
+            <div class="meat"></div>
+            <div class="bottom-bun"></div>
+            </label>
+
+            <div class="nav">
+            <div class="nav-wrapper">
+            <nav>
+                <ul>
+                    <li><a href="index.html">About me</a></li>
+                    <li><a href="resume.html">Resume</a></li>
+                    <li><a href="#">Projects</a></li>
+                    <li><a href="contact.html">Contact</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <main>
+        <div class="title-main">
+            <div class="square">
+                    </div>
+            <h1>Let's talk</h1>
+        </div>
+       <form>
+            <div class="names-form">
+                <div class="first-name-form">
+                    <label for="f">First Name *</label>
+                    <input class="field-input"  type="text" name="firstName" id="f" required>
+                </div>
+
+                <div class="last-name-form">
+                    <label for="ln">Last Name *</label>
+                    <input class="field-input"  type="text" name="lastName" id="ln" required>
+                </div>
+                
+            </div>
+
+            <div class="email-form">
+                <label for="ema">Email *</label>
+                <input class="field-input"  type="email" name="email" id="ema" required>
+            </div>
+
+            <div class="subject-form">
+                <label for="sub">Subject</label>
+                <input class="field-input"  type="text" name="subject" id="sub" >
+            </div>
+
+            <div class="text-form">
+                <label for="msg">Message</label>
+                <textarea class="textarea-input" name="msg" id="msg" rows="3"></textarea>
+            </div>
+
+            <div class="submit-form">
+                <button type="submit">SEND</button>
+            </div>
+
+       </form>
+    </main>
+
+    <footer>
+        <div class="call">
```

Comment:
Can you get these 3 elements (call, write, follow) to line up in a row on small screens?